src/app/containers/Ad/Amp/index.jsx

```diff
@@ -23,7 +23,8 @@ import { ampLeaderboardStyles, ampMpuStyles } from '../utilities/adSlotStyles';
 const AccessDiv = props => <div {...props} />;
 
 const AdSection = styled.section`
-  background-color: ${C_LUNAR_LIGHT};
+  background-color: ${({ pageType }) =>
+    pageType === 'article' ? '#E6E8EA' : C_LUNAR_LIGHT};
```

Comment:
Worth putting a comment to move this colour into a variable when the migration is complete?